_layouts/base.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "govuk/template.njk" %}
+
+{% block head %}
+  <link rel="stylesheet" href="/assets/govuk.css">
+{% endblock %}
+
+{% block pageTitle %}
+  {{- title if title -}}
+{% endblock %}
+
+{% block header %}
+  <header class="govuk-header" role="banner" data-module="govuk-header">
+    <div class="govuk-header__container govuk-width-container app-header__container--pink">
+      <div class="govuk-header__logo">
+        <a href="/" class="govuk-header__link govuk-header__link--homepage">
+          X-GOVUK
+        </a>
+      </div>
+    </div>
+  </header>
+{% endblock %}
+
+{% block footer %}
+  <footer class="govuk-footer " role="contentinfo">
```

Comment:
Could you import `govukFooter` and use that for customisation?

Reply:
Hmm, good point. Maybe we do need to provide custom footer as well as header… although these setting really should be in the upstream component.

Reply:
Yeah, I'll flag this upstream. Not sure how flexible the Design System team might want to be around the header and footer...